content/blog/wazo-platform-c4-overview.md

```diff
@@ -0,0 +1,78 @@
+Title: Wazo C4 platform overview
```

Comment:
I would prefer that we always keep Wazo Platform as a whole with first letter in capital letter. So the title should be: Wazo Platform C4 overview